config/hubs/azure.carbonplan.cluster.yaml

```diff
@@ -4,6 +4,9 @@ kubeconfig:
   file: secrets/azure.carbonplan.yaml
 support:
   config:
+    nvidiaDevicePlugin:
+      azure:
+        enabled: true
```

Comment:
If you add a configuration value for a specific cluster, you should probably have a default value as well so that `{{ if .Values.config.nvidiaDevicePlugin.azure.enabled }}` won't error by having the object `.Values.config.nvidiaDevicePlugin` be null and then saying "can't read azure of null object" or similar.

Reply:
Think I have done this now

Reply:
Yepp! Is there also some schema you need/should update as part of this? I don't remember if there was one schema for the values you can have for the the deployer scripts config.

Reply:
No the schema only goes as deep as the `config` key https://github.com/2i2c-org/infrastructure/blob/c7bbde9b03288b99835225c4da6929fe98b94b4f/config/hubs/schema.yaml#L13-L23

Reply:
Nevermind, you are updating the support charts input values, but that chart doesn't have a schema (even though helm charts can have schemas to be enforced by `helm` btw). I was thinking you added configuration for the deployer scripts configuration itself, which I recall had a schema defined.